e2e/config_test.go

```diff
@@ -107,7 +120,12 @@ var testConfig = func(o *option.Option, installed bool) {
 			gomega.Expect(limaConfigFilePath).Should(gomega.BeARegularFile())
 			cfgBuf, err := os.ReadFile(filepath.Clean(limaConfigFilePath))
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
-			gomega.Expect(cfgBuf).Should(gomega.SatisfyAll(gomega.MatchRegexp(`cpus: \d`), gomega.MatchRegexp(`memory: \dGiB`)))
+
+			var limaCfg limayaml.LimaYAML
+			err = yaml.Unmarshal(cfgBuf, &limaCfg)
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+			gomega.Expect(reflect.ValueOf(limaCfg).FieldByName("CPUs").IsValid()).Should(gomega.BeTrue())
```

Comment:
Just wondering, does the simplified version fail somehow?

```suggestion
			gomega.Expect(reflect.ValueOf(*limaCfg.CPUs).IsValid()).Should(gomega.BeTrue())
```